### ISSUE TITLE
Too much damage on a ability

### ISSUE THREAD

**Issue by Lestrigon78** [WarEmu/WarBugs#14390]
Good day!
Found another problem with SW.
After the 10.06.19 patch, my teammate entered the game on his character. Released in the RVR-zone and inflicted too much damage to the enemy (indicated on the screenshot).
https://pastenow.ru/5S6BR
Then he flew to the capital and began to test the damage on the mannequins, taking off / putting on equipment. The result is the same: increased damage. The result was obtained by the characteristics through the command .getstats. It is seen on the screen that the range power indicator is increased.
After re-entering the game, the bug disappeared and the magnitude of the damage returned to normal.
https://pastenow.ru/5S6BY
https://pastenow.ru/5S6C1

**Comment by sioding** [comment]
Duplicate of #14334

**Comment by Natherul** [comment]
@Lestrigon78 can you please contact me over PM on discord and explain the exact process that resulted in those stats?

**Comment by Kirpich-74** [comment]
https://pastenow.ru/eaeca892c94a659b048b793b3540f4a4
https://pastenow.ru/45bb86c1966c26177529c5c3585f4951
I have the same
After re-entering the game, the bug disappeared and the magnitude of the damage returned to normal

**Comment by wargrimnir** [comment]
could you possibly provide an even smaller screenshot of your stats and .getstats in the screenshot. it's not quite small and useless enough.

**Comment by Kirpich-74** [comment]
as open screenshot double click on it

**Comment by grunbag** [comment]
It happen when power stat goes to negative I’ve just tested

**Comment by Kirpich-74** [comment]
it's the talisman in sentinel's ring. he disappeared and I got -54 meleepower

**Comment by wargrimnir** [comment]
Thanks, that should be helpful to test and track down.

**Comment by Kirpich-74** [comment]
will you give me the Manticore? .)
seriously, I was happy to help.

**Comment by wargrimnir** [comment]
Needs to be fixed first.

**Comment by grunbag** [comment]
It’s not sentinel talisman , I’ve just tested it

**Comment by Kirpich-74** [comment]
-54 this is the sentinel ring. and it just ended when I was not in the game

**Comment by grunbag** [comment]
Ok so it happen when talisman expire

**Comment by Kirpich-74** [comment]
yesterday went to the game. the ring has not been touched since the last time and it was empty. same problem. passed at relog.

**Comment by Natherul** [comment]
fix pending for next patch

**Comment by maksshram** [comment]
Good evening , as I understand the error leading to increased damage found and it was associated with the decoration of sentinel or rather a talisman in it, and is not related to the use of my vulnerabilities of the game . On this please unblock my account . After all, I was the first who faced this error and suffered from it , and I was the first to report it on the bug tracker with the help of my friend Lestr .

Saxarok.

**Comment by sioding** [comment]
The bugtracker is not for appeals. I lock this one here. If you want to appeal a ban - please use the forum.

**Comment by Natherul** [comment]
fixed as of todays patch
